src/ui/DropdownFilter.css

```diff
@@ -0,0 +1,13 @@
+.dropdown-display {
+    float:right;
+    width:150px;
+    height:30px;
+    cursor:pointer;
+    border:solid 2px #DDD;
+}
+
+.caption-display {
```

Comment:
use mendix and bootstrap styling. No need to add custom styling.

Reply:
Done